sidebars.js

```diff
@@ -14,20 +14,32 @@
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  documentationSidebar: [{ type: 'autogenerated', dirName: '.' }],
+  //documentationSidebar: [{ type: 'autogenerated', dirName: '.' }],
 
   // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
-    'intro',
-    'hello',
+
+  documentationSidebar: [
     {
       type: 'category',
-      label: 'Tutorial',
-      items: ['tutorial-basics/create-a-document'],
+      label: 'About Lisk',
+      collapsible: false,
```

Comment:
I think instead of not collapsing tabs at all, we should use this property.
```suggestion
       autoCollapseCategories: true,
```

Reply:
imho we can keep the current content as not collapsible, because we do not have that much content anyway. 
`autoCollapseCategories` is a setting that only makes sense for collapsible categories.

Reply:
Alright, that sounds logical, as we only have three expected categories. Can discuss this in the future, if this needs a change.